source/speech/__init__.py

```diff
@@ -1071,14 +1064,17 @@ def speakTextInfo(
 	)
 
 	if reason == controlTypes.REASON_SAYALL:
-		# Deprecation warning: In 2021.1 speakTextInfo will no longer  send speech through speakWithoutPauses
-		# if reason is sayAll, as sayAllhandler does this manually now.
-		return _speakWithoutPauses.speakWithoutPauses(speechSequences)
+		log.error(

```

Comment:
More explicit about deprecation. Perhaps we should start doing this rather than just a deprecation comment?